### ISSUE TITLE
Search output

### ISSUE THREAD

**Issue by AgenteMacaco** [ror-community/ror-api#266]
hello, 
I couldn't find a query about this so I post a new one. 

I found that, whenever I search an institution using their grid id, the first option proposed doesn't match the grid I run.
For example, if I search for **grid.8756.c** (University of Glasgow), the first institution I find is Genetic Research Institute of the Desert, while Glasgow Uni is the second option. 

Is there a reason why the first option doesn't match my search? 

Many thanks
Ombretta

![Screenshot 2022-10-19 160901](https://user-images.githubusercontent.com/61389401/196730710-0dcabc4f-86f6-4e6b-8129-5550b0695ce5.png)

**Comment by amandafrench** [comment]
Hi Ombretta! That's because the search interface to the ROR registry does a fairly rough and simple text query using the [?query parameter](https://ror.readme.io/docs/rest-api#query-parameter) of the ROR REST API. It's matching the Genetic Research Institute of the Desert because that organization is also known by the acronym GRID, which matches the `grid` part of the string you entered. 

To look specifically for GRID IDs, the best way is to use the [?query.advanced parameter](https://ror.readme.io/docs/advanced-search) and search specifically in the `external_ids` field. Here's how I'd construct it:

https://api.ror.org/organizations?query.advanced=external_ids.%5c*:grid.8756.c

Hope that helps. There might be a way to formulate your search so that it works in the search UI, though -- I'll give it a shot.

**Comment by amandafrench** [comment]
I've played around with different queries in the search interface, but nothing I tried helped the results. The upshot is that entering a GRID ID is pretty much always going to return the Genetic Research Institute of the Desert record first because of the exact match on its GRID acronym, but the 2nd result is always going to be the organization with the GRID ID you searched for. So it mostly works, with one quirk! 

Over on [ror-roadmap](https://github.com/ror-community/ror-roadmap/) I'll suggest optimizing the search so that it prefers the record with the GRID ID as the first record, but I don't know that that change will be a priority. Thanks for the report, @AgenteMacaco!
